deployment/assets/create-script.py

```diff
@@ -2,15 +2,10 @@
 import json
 
 MODELS_LIST = json.loads(os.getenv('MODELS_LIST', ["mini"]))
```

Comment:
Using the default results in this error:
```
    MODELS_LIST = json.loads(os.getenv('MODELS_LIST', ["mini"]))
                  ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/homebrew/Cellar/python@3.11/3.11.4_1/Frameworks/Python.framework/Versions/3.11/lib/python3.11/json/__init__.py", line 339, in loads
    raise TypeError(f'the JSON object must be str, bytes or bytearray, '
TypeError: the JSON object must be str, bytes or bytearray, not list
```

A fix is to put quotes around the default to make it a string like this:
'["mini"]'

Reply:
Thank you!

Reply:
so in addition to the quoting fix, this needs to be a 2 level name according to the following comment